app.py

```diff
@@ -352,15 +352,13 @@ def run_optimization_mip(
         str: Class name for the Classical tab.
         str: The label for the Classical tab.
         bool: True if Classical tab should be disabled, False otherwise.
-        str: Run button class.
-        str. Cancel button class.
         bool: Whether Classical solver is running.
     """
     if ctx.triggered_id != "run-button" or run_click == 0:
         raise PreventUpdate
 
     if SamplerType.MIP.value not in solvers:
-        return (dash.no_update, dash.no_update, "tab", CLASSICAL_TAB_LABEL, True, False)
+        return (dash.no_update, dash.no_update, "tab", CLASSICAL_TAB_LABEL, dash.no_update, False)
```

Comment:
This seems to remove the green bar above the tab. Should "tab" be `dash.no_update` as well?

Reply:
It made the most sense to me to have the colored feedback be a most recent run thing so you could differentiate with what was run the most recently and easily spot what had just finished running, but let me know if you don't think that makes sense